TLM/TLM/Manager/Impl/OptionsManager.cs

```diff
@@ -177,13 +177,9 @@ public class OptionsManager : AbstractCustomManager, IOptionsManager {
 				Options.setRealisticPublicTransport(data[37] == (byte)1);
 			}
 
-            if (data.Length >= 39) {
-                Options.setTurnOnRed(data[38] == (byte)1);
-            }
-            
-            if (data.Length >= 40) {
-	            Options.setTurnOnRedEnabledByDefault(data[39] == (byte)1);
-            }
+			if (data.Length >= 39) {
+				Options.setAllowTurnOnRed(data[38] == (byte)1);
+			}
```

Comment:
This is not correct yet because it would load/save the `allowTurnOnRed` flag from/to the former `turnOnRed` flag.

Reply:
Fixed